### ISSUE TITLE
[FEATURE REQUEST] Overtime/Draw Button

### ISSUE THREAD

**Issue by SebiPeppins** [haxball/haxball-issues#343]
My suggestion is that if can put a button to choose draw or overtime
Ej: Time: 3 min
    Goals: 3
    Overtime
    Draw

**Comment by Raamyy** [comment]
You can make it using the API

**Comment by olegmisar** [comment]
@Raamyy, not everyone can create bot and not everyone wants. This feature should be added.

**Comment by Leo-MapCreator** [comment]
I agree with @olegmisar, haxball is not about just API and some features should work either the room is hosted by API or not
